src/seedu/addressbook/data/person/Address.java

```diff
@@ -36,7 +36,7 @@ public static boolean isValidAddress(String test) {
     }
 
     @Override
-    public String toString() {
+    public String getPrintableString() {
```

Comment:
toString and getPrintableString are actually slightly different in terms of the string representations they return. Pls refer to the documents again to find out and try reimplement it.

Reply:
You should still keep the toString method as it might be used elsewhere